doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -10,6 +10,6 @@
         "#1": "Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
         "#3": "Set \"serverVersion\" to your server version to avoid edge-case exceptions and extra database calls",
-        "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name?charset=utf8mb4&serverVersion=5.7"
+        "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name"
```

Comment:
Would it make sense to mention that one should also not forget to change the `doctrine.yaml` file? And vice-versa?

Reply:
Good idea. I just changed the `#3` comment line to mention `doctrine.yaml`. But I didn't add a note in `doctrine.yaml` about `.env`: it seemed weird, because `DATABASE_URL` is an env variable, so saying "edit it in `.env`" is only true in dev mode.... I wasn't convinced if it should be added there or not.